alvr/server_openvr/cpp/alvr_server/Controller.cpp

```diff
@@ -230,12 +230,12 @@ bool Controller::onPoseUpdate(float predictionS, FfiHandData handData) {
 
     // Note: following the multimodal protocol, to make sure we want to use hand trackers we need to
     // check controllerMotion == nullptr. handSkeleton != nullptr is not enough.
```

Comment:
Comment is outdated

Reply:
I just remove the comment because I think the rust code is already easier to understand